docs/topics/downloader-middleware.rst

```diff
@@ -366,7 +366,7 @@ HttpCacheMiddleware
     This middleware provides low-level cache to all HTTP requests and responses.
     It has to be combined with a cache storage backend as well as a cache policy.
 
-    Scrapy ships with three HTTP cache storage backends:
+    Scrapy ships with two HTTP cache storage backends:
```

Comment:
What about making sure we do not screw it up again?

```suggestion
    Scrapy ships with the following HTTP cache storage backends:
```

Murphy’s law: we will at some point switch to a single storage backend, and forget to change “backends” to “backend”.